doc/src/aboutus.rst

```diff
@@ -24,6 +24,7 @@ Financial and Infrastructure Support
   * in 2012 for 6 students (`GSoC 2012 <https://github.com/sympy/sympy/wiki/Gsoc-2012-report>`_)
   * in 2013 for 7 students (`GSoC 2013 <https://github.com/sympy/sympy/wiki/Gsoc-2013-report>`_)
   * in 2014 for 10 students (`GSoC 2014 <https://github.com/sympy/sympy/wiki/Gsoc-2014-report>`_)
+  * in 2016 for 8 students (`GSoC 2016 <https://github.com/sympy/sympy/wiki/GSoC-2016-report>`_)
```

Comment:
If you are counting James and Rajith that's 9 students. SymPy as a mentoring org had 7 students (originally 8 but one dropped out).